content/02.introduction.md

```diff
@@ -60,7 +61,10 @@ The dsRNA genome is then transcribed and replicated to create viral mRNAs and ne
 
 #### Transmission
 In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
+Other modes of transmission such as through touching surfaces or objects and then touching mucous membranes can also be investigated.
 While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it has been suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies [@doi:10.3390/ijerph15122757].
+Other aspects of transmission to investigate is the relationship of infectiousness or virus shedding to disease period or symptoms and also which types of transmission events are commonly responsible for cases (such as transmission between relatives, nosocomial transmissions, and other possible types of interactions). Some information about these aspects is available for the highly pathogenic coronaviruses SARS (or SARS-CoV-1) and MERS [@doi:10.1016/B978-0-12-385885-6.00009-2][@doi:10.1038/nrmicro.2016.81].
+For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms, it can also possibly be spread by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html] such as people with symptomatic infections at the stage prior to symptoms or the still under investigation possibility of people with asymptomatic infections.
```

Comment:
```suggestion
For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms,  the virus can be transmitted by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html], such as during the pre-symptomatic stage of infection or in people with asymptomatic infections (a presentation that is still under investigation).
```
I'm not sure how best to mention phrase the last bit!

Reply:
maybe for parenthetical clause in your suggestion either what you have since I understood it when I first read it or something like  "(a presentation which sometimes occurs in other infectious diseases but is still being investigated for SARS-CoV-2)" ? I know this way is wordier but if it can be done clearly then it should be ok. (Note I happened to run into a bit of specific articles so far on asymptomatic but I didn't look at it closely enough to tell how hypothetical it was or what type of evidence/data/models they were using; this is in addition to the general mentions I referenced above which just say it's still not known if it can be spread by people who don't show symptoms but they don't specify if they mean pre-symptomatic &/or asymptomatic {though I know they know those are two of the options}].)

Reply:
That sounds good to me! Done!